solr/core/src/java/org/apache/solr/metrics/prometheus/core/SolrCoreSearcherMetric.java

```diff
@@ -47,24 +45,18 @@ public SolrCoreMetric parseLabels() {
   }
 
   @Override
-  public void toPrometheus(SolrPrometheusCoreExporter solrPrometheusCoreRegistry) {
+  public void toPrometheus(SolrPrometheusCoreExporter solrPrometheusCoreExporter) {
```

Comment:
nice.  You also could have simply called it "exporter" or "prometheus".  It might then be so short enough  that the usages become one-liners.  Any way it's fine.

Reply:
Changed to just exporter. It still makes sense, I was always just big on verbosity but shorter names for one-liners is better.